packages/@webex/plugin-meetings/test/unit/spec/meeting/muteState.js

```diff
@@ -361,23 +364,23 @@ describe('plugin-meetings', () => {
 
       it('does not send remote mute for video', async () => {
         // mute
-        await simulateVideoMuteChange(true);
+        await simulateVideoMuteChange(false);
```

Comment:
why are the values still reversed in all these tests in this PR? now that you've fixed the mock stream I don't think you need to change them, so it would be better to not touch them in this PR and keep them as they were, because for example you didn't update all the comments (like in the line above and other places).

Reply:
Done. Reverted these changes wherever required.